From 4816184f6c9588d3c366a67bdbac9179715a80d1 Mon Sep 17 00:00:00 2001
From: "Patil, Rachna" <rachna@ti.com>
Date: Thu, 16 Feb 2012 16:45:13 +0530
Subject: [PATCH 0769/1092] arm: omap: am335x:Add ick data in Hwmod for TSC

Add interface clock information pertaining to Touchscreen
in hwmod data.

Signed-off-by: Patil, Rachna <rachna@ti.com>
---
 arch/arm/mach-omap2/omap_hwmod_33xx_data.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
index 8aa77c7..66c9de6 100644
--- a/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
+++ b/arch/arm/mach-omap2/omap_hwmod_33xx_data.c
@@ -252,6 +252,7 @@ static struct omap_hwmod_addr_space am33xx_adc_tsc_addrs[] = {
 static struct omap_hwmod_ocp_if am33xx_l4_wkup_adc_tsc = {
 	.master		= &am33xx_l4wkup_hwmod,
 	.slave		= &am33xx_adc_tsc_hwmod,
+	.clk		= "adc_tsc_ick",
 	.addr		= am33xx_adc_tsc_addrs,
 	.user		= OCP_USER_MPU,
 };
-- 
1.7.11.2

